tests/RawInputCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Webmozarts Console Parallelization package.
+ *
+ * (c) Webmozarts GmbH <office@webmozarts.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Webmozarts\Console\Parallelization;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+
+#[AsCommand(
+    name: 'raw-input',
+    description: 'A command to demonstrate the RawInput class',
+)]
+final class RawInputCommand extends Command
```

Comment:
What is this command used for?

Reply:
My bad it's not this. This file is entirely the one you created on your branch.